src/dres.c

```diff
@@ -4,16 +4,19 @@
 #include <errno.h>
 #include "iso22133.h"
 #include "dres.h"
+#include "iohelpers.h"
```

Comment:
What's this?

Reply:
No clue. Legacy code that was needed in this file